fgd_cc_contacts/fgd_cc_contacts.py

```diff
@@ -0,0 +1,167 @@
+        "wardhiigleey",
+        "xamar jajab",
+        "xamar weyn",
+        "yaaqshid"
+    ]
+    raw_district_key = "District (Text) - wt_demog_1"
+    coded_district_key = "District (Text) - wt_demog_1_coded"
+    raw_age_key = "Age (Text) - wt_demog_2"
+    coded_age_key = "Age (Text) - wt_demog_2_coded"
+    raw_gender_key = "Gender (Text) - wt_demog_1"
+    coded_gender_key = "Gender (Text) - wt_demog_1_coded"
+    for td in fgd_cc_data:
+        if raw_district_key not in td:
+            td.append_data({
+                raw_district_key: "NA"
+            }, Metadata(user, Metadata.get_call_location(), time.time()))
+
+        if coded_district_key not in td:
+            td.append_data({
+                coded_district_key: "none"
+            }, Metadata(user, Metadata.get_call_location(), time.time()))
+
+        is_mogadishu = td[coded_district_key] in mogadishu_districts
+        td.append_data({
+            "Raw FGD/CC Response": td["Response_1 (Text) - wt_fgd_cc"],
+            "FGD/CC Consented": td[fgd_cc_consent_key],
+            "Raw District": td[raw_district_key],
+            "Coded District":
+                td[coded_district_key] if td[raw_district_key] != Codes.TRUE_MISSING else Codes.TRUE_MISSING,
+            "Coded Age": td[coded_age_key] if td[raw_age_key] != Codes.TRUE_MISSING else Codes.TRUE_MISSING,
+            "Coded Gender": td[coded_gender_key] if td[raw_gender_key] != Codes.TRUE_MISSING else Codes.TRUE_MISSING,
+            "District Mogadishu": "Yes" if is_mogadishu else "No",
+            "FGD/CC Consented and District Mogadishu":
+                "Yes" if is_mogadishu and td[fgd_cc_consent_key] == "Yes" else "No"
+        }, Metadata(user, Metadata.get_call_location(), time.time()))
+
+    # Filter out respondents who aren't from Mogadishu
+    fgd_cc_data = [td for td in fgd_cc_data if td["District Mogadishu"] == "Yes"]
+
+    # Filter out respondents who are definitely younger than 18
+    adults = []
+    for td in fgd_cc_data:
+        try:
+            age = int(td["Coded Age"])
+            if age >= 18:
+                adults.append(td)
+        except:
+            adults.append(td)
+    fgd_cc_data = adults
+
+    # Rename columns for final output
+    for td in fgd_cc_data:
+        td.append_data({
+            "Phone Number": "+{}".format(phone_uuids.get_phone(td["avf_phone_id"])),
+            "Gender": td["Coded Gender"],
+            "Age": td["Coded Age"],
+            "District": "mogadishu" if td["Coded District"] == "mogadisho" else td["Coded District"]
+        }, Metadata(user, Metadata.get_call_location(), time.time()))
+
+    # Shuffle output
+    random.seed(0)
+    random.shuffle(fgd_cc_data)
+
+    # Write json output
+    IOUtils.ensure_dirs_exist_for_file(json_output_path)
+    with open(json_output_path, "w") as f:
+        TracedDataJsonIO.export_traced_data_iterable_to_json(fgd_cc_data, f, pretty_print=True)
+
+    # Set CC data
+    cc_data = fgd_cc_data[:160]
```

Comment:
What does CC stand for? Could you expand in the comment, I know it's used all over the file, but here it has lost all fo the the associated context